### PR TITLE
Update tests and code to add .NET 7 targets.

### PR DESCRIPTION
**Pull request by AraHaan**
Attempt to fix https://github.com/dotnet/command-line-api/issues/1678.

### CODE REVIEW

**Comment by AraHaan** [comment]
The global.json will need to be updated for 7.0.100-preview2.

File: global.json [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {
   "tools": {
-    "dotnet": "6.0.100",
+    "dotnet": "7.0.100-preview.2.22153.17",
     "runtimes": {
       "dotnet": [
-        "5.0.10"
+        "5.0.10",
+        "6.0.3",
```
Comment (AraHaan) [inline global.json:7]:
```suggestion
        "6.0.3"
```

File: global.json [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+        "7.0.0"
```
Comment (AraHaan) [inline global.json:8]:
```suggestion
```

**Comment by adamsitnik** [comment]
Hi @AraHaan 

We have updated to .NET 7 as part of #1981 PR.

Thank you for your contribution!

**Pull request closed without merging**